invite/templates/invite/denied.html

```diff
@@ -3,5 +3,5 @@
 Access Denied
 {% endblock %}
 {% block content %}
-That is an invalid or expired activation code.
+That is an invalid or expired activation code. You may have already registered an account. Try using the "Forgot Password" link on the edit login page to retrieve your credentials via email.
```

Comment:
Should we capitalize "edit"? Or maybe remove it entirely? I think some people don't really see it as it's own thing, but rather as a part of Aubrey.

Reply:
Also, theoretically this app isn't a part of Edit, so to keep it more widely applicable, not referencing the Edit login page at all would be ideal.